lib/project_types/script/layers/infrastructure/script_project_repository.rb

```diff
@@ -173,6 +175,10 @@ def script_config_repository
           end
         end
 
+        def read_input_query
+          ctx.read(INPUT_QUERY_PATH) if ctx.file_exist?(INPUT_QUERY_PATH)
```

Comment:
Just curious, I was assuming `input.graphql` would be required eventually. Is that accurate?

Reply:
For now, I'm thinking it'll stay optional. For some (most?) Script APIs, an input query makes a lot of sense, but I suspect there will be some very simple ones for which the added complexity may not be worth it.

I'm speculating though 🤷. Feel free to chime in.

Reply:
Yeah that makes sense. We can't know for sure yet, I suppose. I was just wondering about the developer experience and if it would complicate things if some APIs use fetch plans, some don't support it, some its optional, etc. But thats a future thing to think about!